defaults/main.yml

```diff
@@ -43,6 +43,13 @@ siteName: "io"
 nodeBase: "{{ siteName }}"
 nodeGpuBase: "gpu"
 
+# Have systemd restart slurm daemons if they fail to start
+slurm_systemd_override_slurmd: True
+slurm_systemd_override_slurmdbd: False
+slurm_systemd_override_slurmctld: False
```

Comment:
I think it's unlikely, yes. Not impossible, of course. :)

That being said, if slurmctld crashes the entire cluster grinds to a halt. So in lieu of HA slurmctld (I'm working on it...), at least trying to restart it automatically if it crashes seems better than nothing, IMHO.